packages/web-test-helpers/package.json

```diff
@@ -1,16 +1,62 @@
 {
-  "name": "web-test-helpers",
-  "private": true,
+  "name": "@ownclouders/web-test-helpers",
+  "version": "0.1.0",
```

Comment:
What about versioning for publishing? Same strategy as for web-pkg and web-client I'd say... what do you think?

Reply:
Yep I agree 👍  Now that you approved the changes, let me create a `0.0.1` to test the CI, then I'll add the correct version number.